wandb/sdk/wandb_init.py

```diff
@@ -13,6 +13,7 @@
 
```

Comment:
Need to handle 
wandb.init(magic=True)
and possibly the extended syntax of
wandb.init(magic=dict(...))